kubernetes/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - kubernetes
 
+1.1.2 / Unreleased
+==================
```

Comment:
will do. since the manifest was already at 1.1.1 on master i thought it was released